valintalaskenta-laskenta-service/src/main/java/fi/vm/sade/valintalaskenta/laskenta/resource/dao/impl/SeurantaDaoImpl.java

```diff
@@ -0,0 +1,531 @@
+package fi.vm.sade.valintalaskenta.laskenta.resource.dao.impl;
+
+import fi.vm.sade.valintalaskenta.laskenta.resource.dao.SeurantaDao;
+import fi.vm.sade.valintalaskenta.laskenta.resource.domain.Laskenta;
+import fi.vm.sade.valintalaskenta.laskenta.resource.dto.*;
+import java.sql.Array;
+import java.time.Instant;
+import java.util.*;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SeurantaDaoImpl implements SeurantaDao {
+  private static final Logger LOG = LoggerFactory.getLogger(SeurantaDaoImpl.class);
+
+  private final JdbcTemplate jdbcTemplate;
```

Comment:
Tässä ei haluta käyttää Spring Data JDBC:tä?

Reply:
Omasta mielestäni näin tuli simppelein ratkaisu.